### DISCUSSION TITLE
Google redirect with queryParams

### DISCUSSION THREAD

**Discussion by radim10** [supabase/supabase#9845] [Questions]
I'm having trouble passing queryParams with Google signIn, using the JS package (1.35.6). I have this signIn function and it works fine, except the queryParams are not being passed. What am I doing wrong?

` const { user, session, error } = await supabase.auth.signIn(
      {
        provider: 'google',
      },
      {
        redirectTo: '/google-redirect',
        queryParams: {
          uid: invitation?.uid,
        },
      }
    )`

**Comment by danielfabriq** [comment]
I am having similar issue
